pkg/controllers/kube-apiserver.go

```diff
@@ -150,7 +150,8 @@ func (s *KubeAPIServer) configure(cfg *config.MicroshiftConfig) error {
 				"quota.openshift.io/ClusterResourceQuota",
 				"quota.openshift.io/ValidateClusterResourceQuota",
 			},
-			"enable-admission-plugins": {},
+			"enable-admission-plugins":              {},
+			"send-retry-after-while-not-ready-once": {"true"},
```

Comment:
Has this option been merged into OpenShift's KAS?

Upstream seems to be targeting 1.26 IIUC: https://github.com/kubernetes/kubernetes/commit/6e3923d0a4f4720d2d9f628eb9c073d2d3ee291a

Reply:
It's a carried patch in openshift/kubernetes. We don't expect it to ever exist upstream, because the usual recommendation for this problem is to run an external load balancer.